app/controllers/ManageOrTrackController.scala

```diff
@@ -44,13 +44,11 @@ class ManageOrTrackController @Inject() (
     directDebitBackendConnector: DirectDebitBackendConnector)(
     implicit
     ec: ExecutionContext)
-
   extends FrontendBaseController(cc) {
 
-  import requestSupport._
-
   def manageOrTrackVrt: Action[AnyContent] =
     actions.securedActionMtdVrnCheckWithoutShutterCheck.async { implicit request: AuthenticatedRequest[_] =>
+      import requestSupport._
```

Comment:
Why can't you  just import at the top?

Reply:
That was my original idea but the implicits that it imports then interfere with other implicits and give a compiler error.